<!-- -*- mode: XML -*- -->

<launch>
    <node name="dynamixel_manager" pkg="dynamixel_controllers" type="controller_manager.py" required="true" output="screen">
        <rosparam>
            namespace: dxl_manager
            serial_ports:
                pan_tilt_port:
                    port_name: "/dev/ttyUSB0"
                    baud_rate: 57600
                    min_motor_id: 1
                    max_motor_id: 25
                    update_rate: 20
        </rosparam>
    </node>
    <!-- Start tilt joint controller -->
    <rosparam file="$(find my_dynamixel_controller)config/multi_turn.yaml" command="load"/>
    <node name="tilt_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port pan_tilt_port
                tilt_controller"
          output="screen"/>
  <node pkg="my_dynamixel_controller" type="dynamixel_publisher" name="motor_controller" output="screen"/>
  <node pkg="motor_cam_tutorial" type="controller" name="cam_controller" output="screen"/>
  <node pkg="motor_cam_tutorial" type="cam_pub" name="cam_pub" output="screen"/>
  <param name="cam_device" value="0" type="int"/>
  <param name="Qtd_Pos" value="10" type="int"/>
</launch>


